hub/api/tests/test_api.py

```diff
@@ -12,57 +12,62 @@
 from hub.client.utils import has_hub_testing_creds
 
 
-def test_persist_local(local_storage):
-    if local_storage is None:
-        pytest.skip()
+# need this for 32-bit and 64-bit systems to have correct tests
+MAX_INT_DTYPE = np.array(100).dtype.name
+MAX_FLOAT_DTYPE = np.array(1.1).dtype.name
```

Comment:
Numpy has [built-in types](https://numpy.org/doc/stable/reference/arrays.scalars.html#built-in-scalar-types) for this purpose. Make sure to test if you accept this commit, because the docs are ambiguous as to how these work for non-linux OSes.
```suggestion
MAX_INT_DTYPE = np.int_.__name__
MAX_FLOAT_DTYPE = np.float_.__name__
```